wagtail/images/models.py

```diff
@@ -459,7 +459,10 @@ def get_transform(self, image):
          - .matrix - An affine transformation matrix that combines any
            transform/scale/rotation operations that need to be applied to the image
         """
-        transform = ImageTransform((image.width, image.height))
```

Comment:
I think this also highlights a minor bug with our image width/height values not accounting for the EXIF orientation, which i think are displayed in the admin.

Anyone think we should solve this too?

This wouldn't change anything for this PR though since we'd obviously still have to deal with legacy images with incorrect width/height

Reply:
I'm not sure if anything needs done for that. It does show the correct width and height of the `original` format rendition in the image admin. That rendition is just created with an incororect width and height on 2.15 if `willow.auto_orient()` had to rotate the image.

You know far more about the internals of this than I do. If there is more to that part than I'm seeing, I'd love to stay in the loop and, time allowing, help with resolving it.

Reply:
Oh, I didn't know we did that! This is fine then, thanks!